views/includes/css.hbs

```diff
@@ -1,4 +1,4 @@
-{{!-- Copyright 2015-2016, Google, Inc.
+/* Copyright 2015-2016, Google, Inc.
```

Comment:
Two reasons: the file contents are CSS; and (2) I used autoprefixer to ensure coverage of all browsers and the HBS comment causes the tool to choke. Cheers.